app/models/user.rb

```diff
@@ -30,8 +30,8 @@ def two_factor_enabled?
     phone.present?
   end
 
-  def send_two_factor_authentication_code(code, options = {})
-    UserOtpSender.new(self).send_otp(code, options)
+  def send_two_factor_authentication_code(_code)
+    # no-op
   end
```

Comment:
Could we add a comment and/or link to this PR? Just want to make it clear how/why we are overriding the typical functionality of the two_factor_authentication gem.